src/MSALWrapper/TokenFetcherPublicClient.cs

```diff
@@ -331,6 +333,7 @@ public async Task<TokenResult> GetTokenNormalFlowAsync(IPCAWrapper pcaWrapper, I
                     {
                         this.ErrorsList.Add(ex);
                         this.logger.LogDebug($"Silent auth failed, re-auth is required.\n{ex.Message}");
+                        this.logger.LogWarning(interactiveAuthMessage);

```

Comment:
```suggestion
                        this.logger.LogWarning($"Opening a browser window for interactive auth.\n{ex.Message}");

```
for consistency.

Reply:
What is the actual text that MSAL comes back with in this scenario? Is it helpful to the user or are we better off with just this simple message?

My thinking here is that the full message seems really useful for debugging, but might be less useful as warning text.

Reply:
Giving it a second thought, we wouldn't need to add this detail to the customer. Simple message should be good.